libs/pbd/pbd/stack_allocator.h

```diff
@@ -61,22 +60,27 @@ class /*LIBPBD_API*/ StackAllocator
 	};
 
 	StackAllocator ()
-		: _ptr ((pointer)&_buf)
+		: _ptr ((pointer)_buf)
 	{ }
 
 	StackAllocator (const StackAllocator&)
-		: _ptr ((pointer)&_buf)
+		: _ptr ((pointer)_buf)
 	{ }
 
 	template <typename U, size_t other_capacity>
 	StackAllocator (const StackAllocator<U, other_capacity>&)
-		: _ptr ((pointer)&_buf)
+		: _ptr ((pointer)_buf)
 	{ }
 
+	StackAllocator& operator= (const StackAllocator&)
+	{
+		return *this;
+	}
+
 	/* inspired by http://howardhinnant.github.io/stack_alloc.h */
```

Comment:
This is incorrect. A copy must not share the underlying memory.
When the copy is modified it will corrupt the memory of the original and vice versa

Reply:
I should have written it, but I did not intend for this to be the final code. I just want to check if by changing the interface to meet that of a copyable type it compiles at all, and if it does, then discuss what the behaviour will be. It's obvious this will not do the right thing.